enviro/templates/enviro/latex_report.tex

```diff
@@ -0,0 +1,10 @@
+{% autoescape on %}

```

Comment:
possibly add a comment that explains why autoescape on is used. (the latex template is loaded by the django template engine which expects html, the "autoescape on" tells it to  treat the following as raw text)